sdk/ml/azure-ai-ml/azure/ai/ml/entities/_assets/_artifacts/_package/model_configuration.py

```diff
@@ -12,7 +12,7 @@
 class ModelConfiguration:
     """ModelConfiguration.
 
-    :param mode: The mode of the model.
+    :param mode: The mode of the model. Possible values include: "Copy", "Download"..
```

Comment:
For this change I would like to see one Unit test on schema load and then compare at the SDK